src/Backend/Core/Layout/Templates/FormLayout.html.twig

```diff
@@ -63,44 +53,18 @@
   {% endif %}
 {% endblock checkbox_radio_label %}
 
-
-{% block help_label_tooltip %}
-  <span class="help-block{% if help_label_tooltip.icon is not same as(false) %} inline{% endif %}">
-    <a href="#" data-toggle="tooltip" data-placement="{{ help_label_tooltip.placement }}" data-title="{{ help_label_tooltip.title|trans({}, translation_domain) }}">
-      {% if help_label_tooltip.icon is not same as(false) %}
-        <i class="fa fa-question"></i>
-      {% endif %}
-      {% if help_label_tooltip.text is not same as(null) %}
-        <span class="text-muted">{{ help_label_tooltip.text }}</span>
-      {% endif %}
-    </a>
-  </span>
-{% endblock help_label_tooltip %}
-
-{% block help_label_popover %}
-  <span class="help-block{% if help_label_popover.icon is not same as(false) %} inline{% endif %}">
-    <a href="#" data-toggle="popover" data-trigger="hover" data-placement="{{ help_label_popover.placement }}" data-title="{{ help_label_popover.title|trans({}, translation_domain) }}" data-content="{{ help_label_popover.content|trans({}, translation_domain) }}">
-      {% if help_label_popover.icon is not same as(false) %}
-        <i class="fa fa-question"></i>
-      {% endif %}
-      {% if help_label_popover.text is not same as(null) %}
-        <span class="text-muted">{{ help_label_popover.text }}</span>
-      {% endif %}
-    </a>
-  </span>
-{% endblock help_label_popover %}
-
-
 {%- block label_asterisk -%}
-  <abbr title="{{ 'lbl.RequiredField'|trans({}, translation_domain)|ucfirst }}">*</abbr>
+  {% import 'Layout/Templates/macros.html.twig' as macro %}
+  {{ macro.required }}
 {%- endblock label_asterisk -%}
 
 {% block form_errors -%}
+  {% import "Layout/Templates/macros.html.twig" as macro %}
```

Comment:
you only have to import it once per file, i see multiple import in this file. I should move the import to the top of the file, out of the block

Reply:
Does that also work for FormLayout files? Because I think I moved it in here because I was getting errors otherwise

Reply:
I not 100% sure, but i never needed to do it the way you did above

Reply:
@carakas could you test this again? I remember having to do something like this for macro's as well, but it was inside of a new macro.

Example:
```
{% macro jobs(jobs) %}
  {% import _self as jobMacros %}
  {% for job in jobs %}
    {% if job.score %}
      {{ jobMacros.job(job.entity, job.score) }}
    {% else %}
      {{ jobMacros.job(job) }}
    {% endif %}
  {% endfor %}
{% endmacro %}
```

Reply:
I tried, if you place it in the file instead of the block you get
```
An exception has been thrown during the rendering of a template ("Notice: Undefined index: macro").
```

Reply:
Very well